.github/workflows/examples-openiotsdk.yaml

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build example - Open IoT SDK
+
+on:
+    push:
+    pull_request:
+    workflow_dispatch:
+
+concurrency:
+    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name == 'pull_request' && github.event.number) || (github.event_name == 'workflow_dispatch' && github.run_number) || github.sha }}
+    cancel-in-progress: true
+
+jobs:
+    openiotsdk:
+        name: Open IoT SDK examples building
+        timeout-minutes: 140
+
+        env:
+            TEST_NETWORK_NAME: OIStest
+
+        runs-on: ubuntu-latest
+        if: github.actor != 'restyled-io[bot]'
+
+        container:
+            image: connectedhomeip/chip-build-openiotsdk:latest
+            volumes:
+                - "/tmp/bloat_reports:/tmp/bloat_reports"
+            options: --privileged
+
+        steps:
+            - uses: Wandalen/wretry.action@v1.0.15
+              name: Checkout
+              with:
+                  action: actions/checkout@v3
+                  with: |
+                      token: ${{ github.token }}
+                  attempt_limit: 3
+                  attempt_delay: 2000
+            - name: Checkout submodules
+              run: scripts/checkout_submodules.py --shallow --recursive --platform openiotsdk
+
+            - name: Detect changed paths
```

Comment:
What do you mean? Which line points to that?

Reply:
It looks to me that builds of various apps depend only if openiotsdk has changed. If that is true, what happens if we change anything else in src ? since API changes may occur, should we not compile then as well? Or if we change /examples/lock-app I would expect to try CI for lock app as well.

Reply:
Sure, I understand the issue. Changed paths detection was removed.